vispy/visuals/colorbar.py

```diff
@@ -255,9 +253,10 @@ class ColorBarVisual(CompoundVisual):
     pos : tuple (x, y)
         Position where the colorbar is to be placed with
         respect to the center of the colorbar
-    label_str : str
+    label : str | vispy.visuals.TextVisual
         The label that is to be drawn with the colorbar
         that provides information about the colorbar.
+        If a TextVisual object then 'label_color' is ignored.
     label_color : str | vispy.color.Color
         The color of the labels. This can either be a
         str as the color's name or an actual instace of a vipy.color.Color
```

Comment:
This also determines the color of the ticks.

Reply:
Ooohhh. :thinking: My added statement isn't exactly wrong though as far as the color of the label goes.